app/src/main/java/org/mozilla/fenix/compose/button/Button.kt

```diff
@@ -86,3 +184,45 @@ private fun PrimaryButtonPreview() {
         }
     }
 }
+
+@Composable
+@Preview
```

Comment:
nit nit: could the previews be in dark mode? I noticed that the rounded corners were hard to notice on the other buttons otherwise.